src/app/newsFeedPage/page.tsx

```diff
@@ -21,26 +22,67 @@ export default function App() {
     getNews();
   }, [news]);
 
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  useEffect(() => {
+    const handleResize = () => setWindowWidth(window.innerWidth);
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
   return (
-    <div className="bg-ivory h-screen">
+    <div className="bg-ivory">
       <NavBar />
-      <div className="p-4">
-        <BackButton />
-        <h1 className="text-night text-3xl font-bold mt-2 -ml-[1.8px]">
-          News
-        </h1>
-        <ul>
-          {news.map(article => (
-            <NewsDisplay
-              key={article.updated_at}
-              id={article.id}
-              contentLink={article.content_link}
-              createdAt={article.created_at}
-              title={article.title}
-            />
-          ))}
-        </ul>
-      </div>
+      {windowWidth < 1024 && (
+        <div>
+          <div className="p-4">
+            <BackButton />
+            <h1 className="text-night text-3xl font-bold mt-2 -ml-[1.8px]">
+              News
+            </h1>
+            <ul>
+              {news.map(article => (
+                <NewsDisplay
+                  key={article.updated_at}
+                  id={article.id}
+                  contentLink={article.content_link}
+                  createdAt={article.created_at}
+                  title={article.title}
+                />
+              ))}
+            </ul>
+          </div>
+        </div>
+      )}
+      {windowWidth >= 1024 && (
+        <div>
+          <div className="ml-[400px] mt-[120px]">
+            <p className="text-night">
+              {' '}
+              <Link href="/" className="text-scary-forest hover:underline">
+                {' '}
+                Home{' '}
+              </Link>{' '}
+              / News{' '}
+            </p>
+            <h1 className="text-night text-4xl font-bold mt-6">News</h1>
+            <div>
+              <ul className="ml-[21.875rem] ">
```

Comment:
nit: can move this to the parent div at line `70` and let child inheritance take care of the rest, better for convention

Reply:
update: Following comment suggestions on line 60, I'd discourage using margins and instead use gap-x tailwind style on the parent div at line 60 to ensure the space between the news displays and the News header is correct